content/en/containers/docker/integrations.md

```diff
@@ -21,70 +21,94 @@ further_reading:
   text: "Assign tags to all data emitted by a container"
 ---
 
+This page covers how to install and configure integrations for your Docker infrastructure by using a Datadog feature known as _Autodiscovery_. This enables you to use [variables][1] like `%%host%%` to dynamically populate your configuration settings. For a detailed explanation of how Autodiscovery works, see [Getting Started with Containers: Autodiscovery][2]. For advanced Autodiscovery options, such as excluding certain containers from Autodiscovery or tolerating unready pods, see [Autodiscovery Management][3].
+
+If you are using Kubernetes, see [Kubernetes and Integrations][4].
+
 <div class="alert alert-info">
-See the <a href="/getting_started/agent/autodiscovery">Autodiscovery Getting Started documentation</a> to discover the concepts behind this feature.
+Some Datadog integrations don't work with Autodiscovery because they require either process tree data or filesystem access: <a href="/integrations/ceph">Ceph</a>, <a href="/integrations/varnish">Varnish</a>, <a href="/integrations/postfix">Postfix</a>, <a href="/integrations/cassandra/#agent-check-cassandra-nodetool">Cassandra Nodetool</a>, and <a href="/integrations/gunicorn">Gunicorn</a>.<br/><br/>
```

Comment:
```suggestion
The following Datadog integrations don't work with Autodiscovery because they require either process tree data or filesystem access: <a href="/integrations/ceph">Ceph</a>, <a href="/integrations/varnish">Varnish</a>, <a href="/integrations/postfix">Postfix</a>, <a href="/integrations/cassandra/#agent-check-cassandra-nodetool">Cassandra Nodetool</a>, and <a href="/integrations/gunicorn">Gunicorn</a>.<br/><br/>
```